tests/pkg/assertions/assertions.go

```diff
@@ -34,6 +34,9 @@ func AssertApplication(t *testing.T, in graphql.ApplicationRegisterInput, actual
 //TODO: After fixing the 'Labels' scalar turn this back into regular assertion
 func AssertLabels(t *testing.T, in graphql.Labels, actual graphql.Labels, app graphql.ApplicationExt) {
 	appNameNormalizier := normalizer.DefaultNormalizator{}
+
+	in["managed"] = false
```

Comment:
This will mess up Nicky Stoykov's test, let's add a check if the label does not exist, and only then add the "false" value